mlibrary/search
# Removing RecordDescription

# Overview
While constructing the [diagram of Search's repository](https://www.figma.com/design/itOuZdXz1c1ATAUE0bFGrR/Search-Repository-Mapping?node-id=6-16&t=gYbuCoxeH8eos4DZ-1), it was discovered that `RecordDescription` did not need its own component. This is because:
1. It only exists in `RecordFull`.
2. The data and imported methods already exist in `RecordFull`.
3. The component only displays a line of code.

This pull request moves the logic into `RecordFull` and deletes `RecordDescription`.

## Anything else?
Code has been simplified using destructuring and only needing to pass necessary information.

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Check a full record and see if anything is broken.
